test/setup.ts

```diff
@@ -1,4 +1,4 @@
-import { configure } from '@superset-ui/translation';
+import { configure } from '@superset-ui/core/src/translation';
```

Comment:
remove `/src/translation`

Reply:
`import { configure } from '@superset-ui'` feels unclear to me. It's not clear what does it configure. Might change it to use the same pattern as `connection/SupersetClient`.

Reply:
But maybe in another PR.